setup.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+
+RED='\033[0;31m'
+NC='\033[0m'
+python_command="python"
+pip_command="pip"
+
+function checkPythonVersion () {
+    case "$($1 --version 2>&1)" in
+    *" 3."*)
+        return 1
+        ;;
+    *)
+        return 0
+        ;;
+    esac
+}
+
+function checkPIPVersion () {
+    case "$($1 --version 6>&1)" in
```

Comment:
I tested without any redirection and it works. Do we want to keep this?
```
p=$(pip --version 6>&1)
pp=$(pip --version)
echo $p # pip 18.1 from ...
echo $pp # pip 18.1 from ...
```

Reply:
Update: search for `python 3` inside `$(pip --version)` instead
something like `[[ $string == *"python 3"* ]] `